src/Cli/dotnet/commands/InstallingWorkloadCommand.cs

```diff
@@ -221,6 +221,18 @@ protected void UpdateWorkloadManifests(WorkloadHistoryRecorder recorder, ITransa
                 if (updateToLatestWorkloadSet)
                 {
                     resolvedWorkloadSetVersion = _workloadManifestUpdater.GetAdvertisedWorkloadSetVersion();
+                    var currentWorkloadVersionInfo = _workloadResolver.GetWorkloadVersion();
+                    if (resolvedWorkloadSetVersion != null && currentWorkloadVersionInfo.IsInstalled && !currentWorkloadVersionInfo.WorkloadSetsEnabledWithoutWorkloadSet)
```

Comment:
How is the current workload version installed, but workload sets are enabled without a workload set? Are those duplicates?

Reply:
If workload sets are enabled but none are installed, then the current workload version will be a `-manifests.<hash>` version, and `IsInstalled` will be true.